skglm/solvers/group_bcd_solver.py

```diff
@@ -4,8 +4,9 @@
 from skglm.utils import AndersonAcceleration, check_group_compatible
 
 
-def group_bcd_solver(X, y, datafit, penalty, w_init=None, Xw_init=None, p0=10,
-                     max_iter=1000, max_epochs=100, tol=1e-4, verbose=False):
+def group_bcd_solver(
```

Comment:
Is there a reason why we don't check intercept optimality here?

Reply:
I forgot it :) Can you implement it?